services/viva-ms/lambdas/submitApplication.js

```diff
@@ -68,7 +68,10 @@ export async function main(event, context) {
 
   if (vivaPostError) {
     throw new TraceException('Failed to submit Viva application', {
-      axios: { ...vivaPostError },
+      errorCode: vivaPostError.status,
```

Comment:
errorCode or statusCode?

Reply:
@petter-a

Reply:
This has been adressed in a later commit